website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -23,15 +22,27 @@ part of the process to [Integrating Kubebuilder and Operator SDK][integration-do
 
 ### What is new
 
-Scaffolded projects now use:
+Projects are now scaffold using:
 
 - [kustomize][kustomize] to manage Kubernetes resources needed to deploy your operator
 - A `Makefile` with helpful targets for build, test, and deployment, and to give you flexibility to tailor things to your project's needs
 - Updated metrics configuration using [kube-auth-proxy][kube-auth-proxy], a `--metrics-addr` flag, and [kustomize][kustomize]-based deployment of a Kubernetes `Service` and prometheus operator `ServiceMonitor`
+- A preliminary support for plugins. For more info see the [extensible CLI and scaffolding plugins design document][plugins-phase1-design-doc]
+- A PROJECT file which stores more information about the resources in use, to enable plugins make useful decisions while scaffolding
+- Liveness and Readiness probes using [`healthz.Ping`][healthz-ping].
```

Comment:
This is a minor feature not really worth mentioning, especially since they are not configurable.
```suggestion
```